Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM  adoptopenjdk/openjdk11:alpine AS build
+WORKDIR /app
+COPY . ./
+RUN ./gradlew clean build
+
+FROM adoptopenjdk/openjdk11:alpine AS unit-test
```

Comment:
Should this be called "test" or something, since I'm guessing the comman would run integration tests too? 🤔